### ISSUE TITLE
Retching

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11684]
# Retching

## Proposed definition

Spasmodic contractions of the diaphragm and abdominal muscles resembling vomiting but without expulsion of gastric contents. PMID:2178746

## Parent term

- `HP:0002017 Nausea and vomiting`

## Synonyms

- Dry heaves
- Nonproductive emesis

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001526
- label: Retching
- parents: [Nausea and vomiting (HP:0002017)](https://hpo.jax.org/app/browse/term/HP:0002017)
- pmid: PMID:2178746
- def: Spasmodic contractions of the diaphragm and abdominal muscles resembling vomiting but without expulsion of gastric contents.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11684
- synonyms:
